src/panels/scene-viewer/SceneViewer.tsx

```diff
@@ -227,11 +232,11 @@ export const SceneViewer = (props: SceneViewerPropsFromParent) => {
       ? (queryParams[getUrlTempVarName(props.options.customSelCompVarName)] as string)
       : undefined;
 
-    let selectedDataBindingResult: Record<string, string> | undefined =
+    let selectedDataBindingResult: ISelectedDataBinding | undefined =
       selectedEntityValue && selectedComponentValue
         ? {
-            [DataBindingLabelKeys.entityId]: selectedEntityValue,
```

Comment:
Just curious why were these changes necessary?

Reply:
This change is required due to the change in the definition for selectedDataBinding in iot-app-kit.